site/gatsby-site/src/components/forms/Tags.js

```diff
@@ -14,13 +14,20 @@ export default function Tags({
   className,
   allowNew = true,
   stayOpen = false,
+  splitChar = ',',
```

Comment:
I think we have a type discrepancy in the field `splitChar`. Here we have a String default value `,` but in the `StepThree.js` file I see that you use a Boolean value.
I suggest to use always String type, and pass a `null` value instead of `false` for these fields.